thinc/backends/ops.py

```diff
@@ -1441,8 +1442,9 @@ def backprop_lstm_gates(
 
 
 def sigmoid(X, out=None):
-    xp = get_array_module(X)
-    return 1.0 / (1.0 + xp.exp(-X))
+    with numpy.errstate(over="ignore"):
+        xp = get_array_module(X)
+        return 1.0 / (1.0 + xp.exp(-X))
```

Comment:
Does this "canonical" safe version throw warnings? (this example is not vectorized)

```
def sigmoid(x):
    if x >= 0:
        z = exp(-x)
        return 1 / (1 + z)
    else:
        z = exp(x)
        return z / (1 + z)
```

Reply:
I don't think one has to branch with vectorization i think its just kinda like:

```
out = zeros_like(X)
out[X >= 0] = 1 / (1 + exp(-x))
out[X < 0] = exp(x) / (1 + exp(x)
```

Reply:
BTW, sorry for the terse answers, typing on my phone ;).